src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -121,29 +123,57 @@ public static boolean isPendingInput(WorkflowRun run) {
     }
 
     /**
+     * Deprecated version that coerces {@link GenericStatus} values to the original set (without {@link GenericStatus#QUEUED}).
+     * Most consumers should switch to {@link #computeChunkStatus2(WorkflowRun, MemoryFlowChunk)} and handle unknown status codes.
+     */
+    @CheckForNull
+    @Deprecated
+    public static GenericStatus computeChunkStatus(@Nonnull WorkflowRun run, @Nonnull MemoryFlowChunk chunk) {
+        GenericStatus newStatusEnum = computeChunkStatus2(run, chunk);
+        return (newStatusEnum == GenericStatus.QUEUED) ? GenericStatus.IN_PROGRESS : newStatusEnum;
```

Comment:
No, the API definition now *mandates* handling of new status codings by API consumers -- that's the key difference.